dist/runtime/components/testimonial-grid.vue

```diff
@@ -75,8 +72,15 @@ export default {
   data() {
     return {
       toggleLimitValue: this.limit,
+      limitValue: this.limit,
     };
   },
+  mounted() {
+    if (window.screen.width < 768) {
```

Comment:
We do have helper functions in Tools to check for breakpoints rather than screen width by px like "Tools.isBelowBreakpoint('md')"

Reply:
Should have made this comment on the nuxt side not on the dist side my bad.